.vale/styles/Silverstripe-cms/Typography.yml

```diff
@@ -0,0 +1,15 @@
+extends: substitution
+message: Use '%s' instead of '%s'.
+level: error
+nonword: true
+ignorecase: true
+swap:
+  '…': '...'
+  '\(c\)': ©
+  '\(tm\)': ™
+  '\(r\)': ®
+  '×': 'x'
+  '[“”„]': '"'
+  '[‘’‚]': "'"
+  '[‹«]': "<"
+  '[›»]': ">"
```

Comment:
Add blank line at end of file

Reply:
Marking resolved since this is now out of scope.